management/management.py

```diff
@@ -176,6 +176,13 @@ def restart_optimizations(optimizations, client):
             client.modify_tasks(operation='restart', base_result=opt.id)
 
 
+def regenerate_optimizations(optimizations, client):
+    for opt in optimizations:
+        if opt.status == 'INCOMPLETE' and (opt.trajectory is not None):
```

Comment:
I am not quite sure what would constitute a satisfying requirement for regeneration, and my gut feeling tells me something will slip through here. Also, it seems like this will try to regenerate tasks that are currently running by a manager. We can possibly be a bit more careful by checking if `final_molecule` is populated, as that is a better indicator that an optimization really finished.

Reply:
Ah, agreed! Will switch out `opt.trajectory` check with a check on `final_molecule`.

Reply:
Done!